pillar/edx/ansible_vars.sls

```diff
@@ -73,23 +73,17 @@
         business_unit=business_unit,
         env=environment,
         purpose=purpose)) %}
-{% set remote_gradebook = salt.vault.read(
-    'secret-{business_unit}/{env}/remote_gradebook'.format(
-        business_unit=business_unit, env=environment)) %}
 {# END VAULT DATA LOOKUPS #}
 
 {# Begin Duplicated Variables #}
 {# multivariate #}
 {% set CMS_DOMAIN = purpose_data.domains.cms %}
-{% set EDXAPP_CMS_ISSUER = "https://{}/oauth2".format(CMS_DOMAIN) %}
```

Comment:
This should stay in the base pillar config since it is actually more useful for sandboxes than residential deployments.